lib/theme_check/corrector.rb

```diff
@@ -36,5 +36,15 @@ def create_default_locale_json(theme)
       theme.default_locale_json = JsonFile.new("locales/#{theme.default_locale}.default.json", theme.storage)
       theme.default_locale_json.update_contents('{}')
     end
+
+    def add_translation_key(theme, key, value)
```

Comment:
Should this be named `add_default_translation_key` considering we're doing it on the default one?

I could see this being expanded in the other direction (you add one to default, but not the others) and we'd probably want `add_translation_key` for that.